tensorflow/lite/delegates/gpu/common/tasks/lstm_test_util.cc

```diff
@@ -67,14 +67,12 @@ absl::Status LstmTest(TestExecutionEnvironment* env) {
       RETURN_IF_ERROR(
           PointWiseNear({7.0 / 15.0, 10.0 / 15.0, 13.0 / 15.0, 16.0 / 15.0},
                         new_state.data, eps))
```

Comment:
You need to put ';' at the end of line.

Reply:
Hi @terryheo, I think the ; is not necessary, I am assuming this is macro from this path
tensorflow/lite/delegates/gpu/common/google/status_macros.h

`
#define RETURN_IF_ERROR(s) \
  {                        \
    auto c = (s);          \
    if (!c.ok()) return c; \
  }
`

If there is error in my understanding, please correct.